crates/katana/storage/provider/tests/fixtures.rs

```diff
@@ -1,24 +1,151 @@
+use std::collections::HashMap;
 use std::sync::Arc;
 
-use katana_primitives::block::BlockHashOrNumber;
+use katana_primitives::block::{
+    BlockHashOrNumber, FinalityStatus, Header, SealedBlock, SealedBlockWithStatus, SealedHeader,
+};
+use katana_primitives::contract::ContractAddress;
+use katana_primitives::state::{StateUpdates, StateUpdatesWithDeclaredClasses};
 use katana_provider::providers::fork::ForkedProvider;
 use katana_provider::providers::in_memory::InMemoryProvider;
+use katana_provider::traits::block::BlockWriter;
+use katana_provider::traits::state::StateFactoryProvider;
 use katana_provider::BlockchainProvider;
+use katana_runner::KatanaRunner;
+use lazy_static::lazy_static;
+use starknet::macros::felt;
 use starknet::providers::jsonrpc::HttpTransport;
 use starknet::providers::JsonRpcClient;
 use url::Url;
 
+lazy_static! {
+    pub static ref FORKED_PROVIDER: (KatanaRunner, Arc<JsonRpcClient<HttpTransport>>) = {
+        let (runner, provider) = katana_runner::KatanaRunner::new().unwrap();
```

Comment:
We could add an option to startup the runner without logs, but default `--json-log` is on. Or we may have a `.gitignore` in prodivers (or other modules/crates/workspace).

Quickest fix for this PR, I would say we can ignore logs in .gitignore.

Reply:
Just re-read the other PR, perhaps this is not relevant in the context of this PR sorry.

Reply:
> We could add an option to startup the runner without logs, but default --json-log is on.

This may be out of scope of this PR but,

> Quickest fix for this PR, I would say we can ignore logs in .gitignore.

I guess we can do this. Just to prevent people from accidentally committing the logs